src/App.js

```diff
@@ -1,23 +1,10 @@
-import logo from './logo.svg';
-import './App.css';
+import Layout from "./components/Layout/Layout";
+import "./App.css";
```

Comment:
Try to use StyledComponents as everybody is using it and it will be good for you to get use to it. But, there is nothing wrong with this approach